datahub-graphql-core/src/main/resources/timeline.graphql

```diff
@@ -0,0 +1,146 @@
+extend type Query {
+    """
+    Get list of schema blame
+    """
+    listSchemaBlame(input: ListSchemaBlameInput!): ListSchemaBlameResult
+}
+
+type TimelineConfig {
+    """
+    Whether the timeline feature is enabled.
+    """
+    enabled: Boolean!
+}
+
+"""
+Enum of SemanticVersionChange
+"""
+enum SemanticVersionChangeType {
+    NONE
+    PATCH
+    MAJOR
+    MINOR
+    EXCEPTIONAL
+}
+
+
+"""
+Enum of types of changes
+"""
+enum ChangeOperationType {
+    ADD
+    MODIFY
+    REMOVE
+}
+
+"""
+Enum of CategoryTypes
+"""
+enum ChangeCategoryType {
+    DOCUMENTATION
+    GLOSSARY_TERM
+    OWNERSHIP
+    TECHNICAL_SCHEMA
+    TAG
+}
+
+"""
+Input for getting list of latest schema changes computed at a specific version.
+"""
+input ListSchemaBlameInput {
+    """
+    The dataset urn
+    """
+    datasetUrn: String!
+    """
+    Categories to filter on
+    """
+    categories: [ChangeCategoryType!]
+    """
+    Changes after version cutoff are not shown
+    """
+    versionCutoff: String
+
+}
+
+"""
+List of latest schema changes computed at a specific version.
+"""
+type ListSchemaBlameResult {
+    """
+    Latest and current semantic version
+    """
+    latestSemVer: SemVerStruct!
+    """
+    Selected semantic version
+    """
+    selectedSemVer: SemVerStruct
+    """
+    All semantic versions
+    """
+    semVerList: [SemVerStruct!]
+    """
+    List of schema histories
+    """
+    schemaBlameList: [SchemaBlame!]
+}
+
+"""
+Schema history
+"""
+type SchemaBlame {
```

Comment:
"Blame" is not really an "object type". What can we name this that is more clear?

Isn't this more like a "SchemaFieldChange"?